CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - `sandstone/Icon` public class name `icon`
 
+### Fixed
+
+- `sandstone/WizardPanels` to allow preventDefault on `onTransition` and `onWillTransition` to prevent focus assignment
```

Comment:
```suggestion
- `sandstone/WizardPanels` to provide a way to prevent focusing on Panel again by allowing preventDefault when `onTransition` and `onWillTransition`
```